src/query/test/promql/m3comparator_client.go

```diff
@@ -18,7 +18,7 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package test
+package promql
```

Comment:
Hmm not too sure about the package name here, it's not really promql but more comparator stuff

Reply:
`compatibility`tests package?

Reply:
Yeah, `compatibility` sounds just about right.